pom.xml

```diff
@@ -374,7 +374,7 @@
               <excludes>**/*$logger.java,**/*$bundle.java</excludes>
               <useFile/>
               <!-- checkstyle is set to be run on particular modules by disabling skip -->
-              <skip>true</skip>
+              <skip>${checkstyle.skip}</skip>
```

Comment:
If the user does not provide a value for checkstyle.skip this defaults to true (at least it did when I tested it).

Reply:
Please, give me some points how I can reproduce the test failure you can see.

Reply:
Hmm, I cannot reproduce it now, not sure what I did before. Sorry about that.
But good call about using the plugin property name (namely checkstyle.skip)

Reply:
Thank you. I'm going to merge this.